cluster/src/test/java/org/apache/iotdb/cluster/server/member/MemberTest.java

```diff
@@ -190,6 +195,9 @@ public void tearDown() throws Exception {
     ClusterDescriptor.getInstance().getConfig().setRaftLogBufferSize(preLogBufferSize);
     ClusterDescriptor.getInstance().getConfig().setUseAsyncApplier(prevUseAsyncApplier);
     IoTDBDescriptor.getInstance().getConfig().setEnableWal(prevEnableWAL);
+
+    RaftServer.setSyncLeaderMaxWaitMs(20 * 1000);
+    RaftServer.setHeartBeatIntervalMs(1000L);
```

Comment:
It's better to use the saved configuration. We can't guarantee that the default configuration will not be modified.

Reply:
Thanks 
done